packages/ui/src/primitives/tailwind/Input/index.tsx

```diff
@@ -79,7 +79,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
 
     const twClassname = twMerge(
       'text-base font-normal tracking-tight',
-      'textshadow-sm flex h-9 w-full rounded-lg border border-theme-primary bg-theme-surfaceInput px-3.5 py-5 transition-colors',
+      'textshadow-sm flex h-9 w-full rounded-lg border border-theme-primary bg-theme-surfaceInput px-2 py-5 transition-colors',
```

Comment:
In [native designs](https://www.figma.com/design/Gn8EScrMmg1g6sQBy8f6lz/Hi-FI%3A-Dashboard-%26-Admin?node-id=863-101448&t=CmLbdDLbokTEUFtr-0), the padding is actually 14px
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/003a96e1-0568-47cb-8e09-72a36149d669)

Reply:
seems like the image above is from the admin dashboard. This pr is for the studio

Reply:
just want to confirm, the image above is for the admin dashboard. the spacing in the design shows 8px
<img width="613" alt="Screenshot 2024-06-03 at 12 33 32 PM" src="https://github.com/EtherealEngine/etherealengine/assets/44120813/4f493ee7-2377-4e8f-897e-e8287d40b146">